### PR TITLE
A muxshark blog post

### PR DESCRIPTION
**Pull request by kevinoliver**
This is a short blog post introducing our Wireshark detector for the Mux protocol.

### CODE REVIEW

File: source/blog/2016-05-19-mux-shark.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+layout: post
+title: Muxshark a Wireshark plugin for the Mux protocol
+published: true
+post_author:
+  display_name: Kevin Oliver
+  twitter: kevino
+tags: Finagle, Mux
+---
+
+This [Wireshark](https://www.wireshark.org/) dissector provides basic decoding of all
```
Comment (cacoco) [inline source/blog/2016-05-19-mux-shark.md]: "The Wireshark dissector"?

Comment (kevinoliver) [inline source/blog/2016-05-19-mux-shark.md]: Yeah they are called dissectors in Wireshark parlance. So I _think_ "This" is correct?

File: source/blog/2016-05-19-mux-shark.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+* Frame length
+* Message type
+* Tag number
+
+> <sub>Don't know about [Wireshark](http://jvns.ca/blog/2016/03/16/tcpdump-is-amazing/)
+or [Mux](http://twitter.github.io/finagle/guide/Protocols.html#mux)?
+Follow the links for introductions.</sub>
+
+While this was released as part of Finagle 6.33.0 we never got around to properly announcing it. Now that Shark Week 2016 is about a month away, it felt like a good time to tell people about it.
```
Comment (taylorleese) [inline source/blog/2016-05-19-mux-shark.md]: +1 for the shark week reference although i'm a bit disappointed there are no shark images in here.

**Comment by taylorleese** [comment]
🐑 🇮🇹

**Comment by mosesn** [comment]
Can we layer mux with thrift?  If so, might be worth calling it out.

**Comment by kevinoliver** [comment]
@mosesn nope, we don't have thrift deserialization for this.

**Comment by mosesn** [comment]
LGTM

**Pull request merged**